src/components/events/upcoming-events/upcoming-events.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import LargeEventsCard from '../large-events-card/large-events-card';
+import events from '../../../data/events';
+import { isToday, isPastDate } from '../../../utils/date-utils';
+import './upcoming-events.css';
+
+const UpcomingEvents = () => {
```

Comment:
Not important to do right now, but would be a good thing in the future to export those functions into the utils folder if we plan on reusing them :)